tests/common/dualtor/dual_tor_utils.py

```diff
@@ -557,3 +557,13 @@ def check_tunnel_balance(ptfhost, active_tor_mac, standby_tor_mac, active_tor_ip
                 log_file=log_file,
                 qlen=2000,
                 socket_recv_size=16384)
+
+
+def crm_nexthop_counter(host):
```

Comment:
suggest name this function as get_crm_nexthop_counter

Reply:
Updated. Thanks!